engine/withdraw_test.go

```diff
@@ -149,12 +149,9 @@ func TestParseEvents(t *testing.T) {
 		if x%2 == 0 {
 			resp.RequestDetails.Currency = currency.AUD
 			resp.RequestDetails.Type = 1
-			resp.RequestDetails.Fiat = new(withdraw.FiatRequest)
```

Comment:
Yeah, I can leave this PR open for a little bit and add in any checks that's appropriate. I Will add this, just let me know if there is anything else you want to be added.

Reply:
Awesome!

That was the only one i noticed it also left them empty in master thought it would be just good to have some data there for if its ever changed in future